bot/code_review_bot/report/phabricator.py

```diff
@@ -191,9 +198,11 @@ def publish(self, issues, revision, task_failures, notices, reviewers):
             if patch.analyzer.name not in self.analyzers_skipped
         ]
 
-        if issues:
+        # Publish a statistic about issues already known on mozilla-central
+        known_issues = [issue for issue in issues if issue.new_issue is False]
+        if publishable_issues:
             # Publish detected patch's issues on Harbormaster, all at once, as lint issues
-            self.publish_harbormaster(revision, issues)
+            self.publish_harbormaster(revision, publishable_issues)
```

Comment:
Doesn't this mean publishing all issues, even previously existing ones?

Reply:
This is related to https://github.com/mozilla/code-review/pull/1581#discussion_r1183178491

I moved the publication logic (depending on before/after) to `Issue.is_publishable()` so it is much cleaner now : issues inside the patch OR unknown to the backend.